lua/simfphys_darkrp_modules/fuelprice/modules/taxes/server/sv_main.lua

```diff
@@ -37,6 +37,22 @@ hook.Add( "Slawer.UpdateTaxes", "CheckForUpdatedFuelTaxes", function( ply, taxDa
     FuelPrices:UpdateTaxRates( taxData.fuelTaxes )
 end )
 
+hook.Add( "SimfPhys_FuelTaxes_ChargedCustomer", "SendTaxesToSlawer", function( pump, customer, priceStruct )
+    local taxAmount = pump:GetNWFloat( "FuelTax" )
+    local price = priceStruct.price or 0
+    local taxRevenue = price / ( 1 + taxAmount )
```

Comment:
Assuming the price passed into this function is post tax, tax revenue here will actually be price pre tax. I think you need `price - ( price / (taxAmount + 1 ) )` or something like that.

Reply:
Ah, yep